tests/foreman/cli/test_errata.py

```diff
@@ -639,6 +643,186 @@ def test_positive_list_affected_chosts_by_erratum_restrict_flag(self):
             }
         )
 
+    @tier3
+    def test_host_errata_search_commands(self):
+        """View a list of affected hosts for security (RHSA) and bugfix (RHBA) errata,
+        filtered with errata status and applicable flags. Applicability is calculated using the
+        Library, but Installability is calculated using the attached CV, and is subject to the
+        CV's own filtering.
+
+        :id: 07757a77-7ab4-4020-99af-2beceb023266
+
+        :Setup: Errata synced on satellite server.
+
+        :Steps:
+            1.  host list --search "errata_status = errata_needed"
+            2.  host list --search "errata_status = security_needed"
+            3.  host list --search "applicable_errata = RHBA-2012:1030"
+            4.  host list --search "applicable_errata = RHSA-2012:0055"
+            5.  host list --search "applicable_rpms = kangaroo-0.3-1.noarch"
+            6.  host list --search "applicable_rpms = walrus-5.21-1.noarch"
+            7.  Create filter & rule to hide RPM (applicable vs. installable test)
+            8.  Repeat steps 3 and 5, but 5 expects host name not found.
+
+        :expectedresults: The hosts are correctly listed for RHSA and RHBA errata.
+
+        """
+        # Install package on first VM to create a need for RHBA-2012:1030
+        result = ssh.command(
+            f'yum install -y {FAKE_4_CUSTOM_PACKAGE}', self.virtual_machines[0].ip_addr
+        )
+        assert result.return_code == 0, "Failed to install RPM"
+        # Update package on first VM to remove its need for RHSA-2012:0055
+        result = ssh.command(
+            f'yum install -y {FAKE_2_CUSTOM_PACKAGE}', self.virtual_machines[0].ip_addr
+        )
+        assert result.return_code == 0, "Failed to install RPM"
```

Comment:
Good to reduce the line of code using iteration over the package list or tuple

Reply:
Hello @omkarkhatavkar , as its only the first VM that gets the packages, not sure how to change that part to use iteration over the package list and still make it easy to understand and maintain. I could just use one `yum install` line, then need to modify and move two lines of comments. Changes made. Please review, thank you.